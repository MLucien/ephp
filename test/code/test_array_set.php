<?php

$a = [];
$a[] = 1;
$a[] = 2;
$a[] = 3;
print_r($a);

for ($i=0; $i<3; $i++) {
	$a[$i] *= $a[$i];
	$a[$a[$i] * 10] = $a[$i];
	print "{$a[$i]} ";
}
print "\n";

foreach ($a as $k => $v) {
	print "$k => $v\n";
}

