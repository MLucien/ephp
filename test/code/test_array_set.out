Array
(
    [0] => 1
    [1] => 2
    [2] => 3
)
1 4 9 
0 => 1
1 => 4
2 => 9
10 => 1
40 => 4
90 => 9
